config/rbac/leader_election_role.yaml

```diff
@@ -9,21 +9,19 @@ rules:
   resources:
   - configmaps
   verbs:
+  - create
```

Comment:
@yongja79 Similar to WLO, changes to leader election should be reversed. Thanks.